lib/state/simperium/actions.ts

```diff
@@ -9,3 +9,7 @@ export const remoteNoteUpdate: A.ActionCreator<A.RemoteNoteUpdate> = (
   noteId,
   note,
 });
+
+export const reallyLogOut: A.ActionCreator<A.ReallyLogOut> = () => ({
+  type: 'REALLY_LOG_OUT',
+});
```

Comment:
as an action this feels more like a UI action than a Simperium one. is there a reason to have it in this file?

Reply:
Resolved in https://github.com/Automattic/simplenote-electron/pull/2277/commits/973f4d9f0dd18b1bf5467297f617df1bac2ccbf2